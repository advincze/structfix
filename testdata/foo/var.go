package p

var _ = foo{bar: {baz: {qux: {quux: 42}}}}
