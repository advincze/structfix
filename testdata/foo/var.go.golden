package p

var _ = foo{bar: struct {
	baz struct {
		qux struct {
			quux int
		}
	}
}{baz: struct {
	qux struct {
		quux int
	}
}{qux: struct {
	quux int
}{quux: 42}}}}
